gordo_components/builder/build_model.py

```diff
@@ -68,6 +71,14 @@ def build_model(
                 Example::
 
                     {"cv_mode": "cross_val_only"}
+            - scoring_scaler: Optional[str]
```

Comment:
think this should go above the example, and maybe update the example with this as well?